runtime/src/main/java/org/corfudb/runtime/collections/SMRMap.java

```diff
@@ -2,6 +2,8 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSet;
+import com.sun.javafx.collections.MappingChange;
+
 import java.util.Collection;
```

Comment:
I think this will throw a class not found exception when you run in jre headless mode. You're also not using it, remove it?